modules/tpmnBidAdapter.js

```diff
@@ -1,126 +1,222 @@
 /* eslint-disable no-tabs */
 import { registerBidder } from '../src/adapters/bidderFactory.js';
-import { parseUrl, deepAccess } from '../src/utils.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
 import { getStorageManager } from '../src/storageManager.js';
-import { BANNER } from '../src/mediaTypes.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { Renderer } from '../src/Renderer.js';
 import { config } from '../src/config.js';
+import * as utils from '../src/utils.js';
 
-export const ADAPTER_VERSION = '1';
-const SUPPORTED_AD_TYPES = [BANNER];
 const BIDDER_CODE = 'tpmn';
-const URL = 'https://ad.tpmn.co.kr/prebidhb.tpmn';
-const IFRAMESYNC = 'https://ad.tpmn.co.kr/sync.tpmn?type=iframe';
-export const storage = getStorageManager({bidderCode: BIDDER_CODE});
+const DEFAULT_BID_TTL = 500;
+const DEFAULT_CURRENCY = 'USD';
+const SUPPORTED_AD_TYPES = [BANNER, VIDEO];
+// const BIDDER_ENDPOINT_URL = 'http://localhost:8081/ortb/pbjs_bidder';
+const BIDDER_ENDPOINT_URL = 'https://gat.tpmn.io/ortb/pbjs_bidder';
+const IFRAMESYNC = 'https://gat.tpmn.io/sync/iframe';
+const VIDEO_ORTB_PARAMS = [
+  'mimes',
+  'minduration',
+  'maxduration',
+  'placement',
```

Comment:
you're missing plcmt

Reply:
[792fc2c](https://github.com/prebid/Prebid.js/pull/10611/commits/792fc2ce6e74debec8b342030393405f68ac9438)
add plcmt

[8bf7567](https://github.com/prebid/Prebid.js/pull/10611/commits/8bf75676b74ca461f4cf76b884bf99c8e720af52)
instream video is sound on default, outstream video sound off default